tests/UnifiedSpecTests/command-monitoring/writeConcernError.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
When syncing spec tests, please include a reference to the specs commit. For this PR, it should be:

```
Synced with mongodb/specifications@18932c00f9e2f8be66b48c930cbbc428499db668
```

This makes it easier to jump to the specs repo when viewing PHPLIB's git history.